CONTRIBUTING.md

```diff
@@ -58,31 +58,78 @@ maps), `time/`, `os/`, etc. Their documentation is pretty clear.
 username will be referred to later as 'YOUR_GITHUB_USERNAME'. Change it
 accordingly in the steps below.)
 
-1. Clone https://github.com/vlang/v in a folder, say nv (`git clone
-https://github.com/vlang/v nv`)
-1. `cd nv`
-1. `git remote add pullrequest git@github.com:YOUR_GITHUB_USERNAME/v.git`  #
-(NOTE: this is your own forked repo of: https://github.com/vlang/v - After
-this, we just do normal git operations such as: `git pull` and so on.)
-1. When finished with a feature/bugfix, you can: `git checkout -b fix_alabala`
-1. `git push pullrequest`  # (NOTE: the pullrequest remote was setup on step 3)
-1. On GitHub's web interface, I go to: https://github.com/vlang/v/pulls  Here
-the UI shows a nice dialog with a button to make a new pull request based on
-the new pushed branch. (Example dialogue:
-https://url4e.com/gyazo/images/364edc04.png)
-1. After making your pullrequest (aka, PR), you can continue to work on the
-branch... just do step #5 when you have more commits.
-1. If there are merge conflicts, or a branch lags too much behind V's master,
+1. Fork https://github.com/vlang/v using GitHub's interface in your own account, lets say that the forked repository is at
```

Comment:
"to your own account"?

Reply:
Thanks. Changed.

Reply:
"Let's say..."

And it should also be in a new sentence.